code/modules/vehicles/mecha/_mecha.dm

```diff
@@ -292,6 +296,15 @@
 	icon_state = get_mecha_occupancy_state()
 	return ..()
 
+/obj/vehicle/sealed/mecha/proc/set_mouse_pointer()
+	if(weapons_safety)
+		mouse_pointer = ""
+	else if(equipment_disabled)
+		mouse_pointer = 'icons/effects/mouse_pointers/mecha_mouse-disable.dmi'
+	else mouse_pointer = 'icons/effects/mouse_pointers/mecha_mouse.dmi'
```

Comment:
Cursed syntax?

Reply:
Hopefully what I have now is less cursed.